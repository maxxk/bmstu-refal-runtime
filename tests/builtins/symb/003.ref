$ENTRY Go
{
	= <Prout <Symb 'sdasd'>>
}

