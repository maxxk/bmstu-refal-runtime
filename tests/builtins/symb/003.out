[Builtins][Symb]: Symb takes 1 argument - int number!
